mobile/index.js

```diff
@@ -1,6 +1,8 @@
 import { AppRegistry } from 'react-native';
+import TrackPlayer from 'react-native-track-player';
 
 import { name as appName } from './app.json';
 import { App } from './src/libs/components/components';
 
 AppRegistry.registerComponent(appName, () => App);
+TrackPlayer.registerPlaybackService(() => require('./service.js'));
```

Comment:
Won't it work without this handler?

or what if we try to pass empty function here 
```suggestion
TrackPlayer.registerPlaybackService(() => async () => {});
```